index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Potfolio</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <header>
+        <nav class="nav">
+            <img class="img" src="images/image_geometry_1.svg" alt="clip art">
```

Comment:
- Good job done so far, but since this image is not part of the main content of the page, it is advisable you add it as a background image. Please fix this 🙏 
- Your navbar does not look exactly like the Figma design template, Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏

Your project:
<img width="280" alt="rhs5hs5" src="https://user-images.githubusercontent.com/73607512/220185423-9caf9f32-386c-4e62-93b9-a5edffb4e9cc.PNG">

Design template:
<img width="288" alt="jgm n, n" src="https://user-images.githubusercontent.com/73607512/220186014-fdf370a5-0210-4476-9af5-7abb6b0adbdb.PNG">

Reply:
> * Good job is done so far, but since this image is not part of the main content of the page, it is advisable you add it as a background image using CSS. Please fix this.
> * Your navbar does not look exactly like the Figma design template, Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements pray.

- [ ] As the previous reviewer suggested, kindly implement those. :+1: 

Your design             |  Figma Design
:-------------------------:|:-------------------------:
![yourdesign](https://user-images.githubusercontent.com/70483835/220191182-0351daa6-3620-4711-a0fe-ae5624bb7b2c.png) |  ![template](https://user-images.githubusercontent.com/70483835/220191211-1dab5435-9da8-47cc-8be2-4d4b6947be7e.png)